modules/virt-enabling-disabling-vm-delete-protection-cli.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-cli_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the command line
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the command line. Should you change your mind, you can easily disable delete protection for a VM.
+
+By default, VMs are not enabled with delete protection. You must set the option for each individual VM.
```

Comment:
I don't think this reads right. Added a suggestion below but I'm sure you could do it another way

```suggestion
By default, delete protection is not enabled for VMs. You must set the option for each individual VM.
```